site/content/docs/5.3/extend/icons.md

```diff
@@ -10,181 +10,225 @@ toc: true
 
 While Boosted doesn't include an icon set by default, Orange does have its own comprehensive icon library called Solaris. While most icon sets include multiple file formats, we prefer SVG implementations for their improved accessibility and vector support.
 
-## Solaris
-
-Solaris is a growing library of SVG icons that are designed by [Orange's Global Design Language Team](https://design.orange.com/).
-
-They are not open-source though and should only be used for Orange branded projects. Please refer to our [`NOTICE.txt` file for legal information]({{< param repo >}}/blob/v{{< param current_version >}}/NOTICE.txt).
-
-[Learn more about Solaris]({{< param icons >}}).
+[Solaris icons library]({{< param icons >}}) is a growing library of icons that are designed by [Orange's Group Design Team](https://system.design.orange.com/).
+It features an icon search tool based on categories and keywords, and allows icons to be downloaded in various formats for use in design and development.
+Please refer to the Solaris Icon Library user documentation for full design and development guidelines.
+
+They are not open-source though and should only be used for Orange branded projects. Please refer to the [icons license file]({{< param icons_license >}}) for legal information.
+
+## Use Solaris icons
+
+There are many ways to use Solaris icons in a web page:
+- Bundled, if you have several icons in your project:
+  - [SVG sprite]({{< docsref "extend/icons#svg-sprite" >}}) (preferred choice for flexibility, performance and accessibility)
+  - [Web font]({{< docsref "extend/icons#web-font" >}})
+- Individually, if you have only few icons to render:
+  - [Inline SVG]({{< docsref "extend/icons#inline-svg" >}})
+  - [SVG external image]({{< docsref "extend/icons#svg-external-image" >}})
+  - [CSS background SVG]({{< docsref "extend/icons#css-background-svg" >}})
+
+See Solaris icons library [develop documentation]({{< param icons_doc >}}) for complete information about the different formats and their pros and cons.
```

Comment:
The link doesn't go to the right page. Plus, we could say "development" instead of "develop"; it sounds weird to me.